CONTRIBUTING.md

```diff
@@ -4,27 +4,34 @@ As mentioned in the [README](https://github.com/haskell/containers/blob/master/R
 
 For proposing API changes/enhancements, please follow the [guidelines outlined on the Haskell Wiki](https://wiki.haskell.org/Library_submissions#Guide_to_proposers). Especially note that all API changes/enhancements should be discussed on libraries@haskell.org mailing list.
 
-## Building and testing
+## Building, testing, and benchmarking
 
-Building and testing the containers package can be done using either `cabal-install` or `stack`.
+Building, testing, and benchmarking the containers package can be done using either `cabal-install` or `stack`.
```

Comment:
I suggest you mention the fact that the procedures here need to be a bit weird. If you have a good understanding of why (I don't) that might be good to mention.

Reply:
TL;DR; `cabal configure` isn't able to solve the constraints (I haven't had time to investigate why yet). Here's the error message:

```
Resolving dependencies...
Warning: solver failed to find a solution:
Could not resolve dependencies:
rejecting: containers-0.5.10.2:!bench (constraint from config file, command
line flag, or user target requires opposite flag selection)
trying: containers-0.5.10.2:*bench
unknown package: random (dependency of containers-0.5.10.2:*bench)
Dependency tree exhaustively searched.
Trying configure anyway.
Configuring containers-0.5.10.2...
cabal: Encountered missing dependencies:
ChasingBottoms -any,
HUnit -any,
QuickCheck >=2.7.1,
criterion >=0.4.0 && <1.3,
random <1.2,
test-framework >=0.3.3,
test-framework-hunit -any,
test-framework-quickcheck2 >=0.2.9
```

I plan on investigating this at some point. I've updated the paragraph with a brief explanation of why we don't just use `cabal configure`.

Reply:
Filed https://github.com/haskell/containers/issues/462.